airflow/models/connection.py

```diff
@@ -233,7 +233,14 @@ def get_password(self) -> Optional[str]:
                     f"Can't decrypt encrypted password for login={self.login}  "
                     f"FERNET_KEY configuration is missing"
                 )
-            return fernet.decrypt(bytes(self._password, 'utf-8')).decode()
+            try:
+                decrypted_password = fernet.decrypt(bytes(self._password, 'utf-8')).decode()
+                return decrypted_password
+            except Exception as e:
+                log.exception(str(e))
```

Comment:
Do not do this.

https://docs.python.org/3/library/logging.html#logging.exception

> `logging.exception(msg, *args, **kwargs)`
>
> Logs a message with level `ERROR` on the root logger. The arguments are interpreted as for `debug()`. Exception info is added to the logging message. This function should only be called from an exception handler.

So this logs the exception message_twice`, once because `exception()` adds it, once because you pass it. You should pass a user-friendly message instead, and let `exception()` log the exception for you.